backend/hitas/tests/factories/indices.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+
+from factory import fuzzy
+from factory.django import DjangoModelFactory
+from faker import Faker
+
+from hitas.models import (
+    ConstructionPriceIndex,
+    ConstructionPriceIndexPre2005,
+    MarketPriceIndex,
+    MarketPriceIndexPre2005,
+    MaxPriceIndex,
+)
+
+faker = Faker(locale="fi_FI")
+
+
+class AbstractIndexFactory(DjangoModelFactory):
+    class Meta:
+        django_get_or_create = ("value",)
+        abstract = True
+
+    month = fuzzy.FuzzyDate(start_date=datetime.date(1990, 1, 1))
+    value = fuzzy.FuzzyDecimal(low=0)
```

Comment:
Should this be `django_get_or_create = ("month",)` instead?
There may be a case where the same value for month is generated, which would cause a duplicate pk exception

Reply:
fixed by removing it, it was unnecessary